db/python/tables/sequencing_group.py

```diff
@@ -556,3 +556,32 @@ async def get_type_numbers_for_project(self, project) -> dict[str, int]:
         """
         rows = await self.connection.fetch_all(_query, {'project': project})
         return {r['type']: r['n'] for r in rows}
+
+    async def get_type_numbers_history(
```

Comment:
This returns data at a bit of a funny level of granularity, a method that returns a list of days that had sequencing groups created and the count of sgs created on that day is always going to need some sort of transformation for the data to be useful. It would make sense to me to move the data transformation from the layer into this method so that the result from this method is a bit more useful. 

As a general rule, table methods are intended for getting data from the database and shouldn't contain much business logic or things like permission checks, but data transformations to get the data into a useful shape are fine to have in here.